src/patternfly/components/Chip/chip.scss

```diff
@@ -79,22 +82,28 @@
     }
   }
 
-  // Button
-  .pf-c-button {
-    --pf-c-button--PaddingTop: var(--pf-c-chip__c-button--PaddingTop);
-    --pf-c-button--PaddingRight: var(--pf-c-chip__c-button--PaddingRight);
-    --pf-c-button--PaddingBottom: var(--pf-c-chip__c-button--PaddingBottom);
-    --pf-c-button--PaddingLeft: var(--pf-c-chip__c-button--PaddingLeft);
-    --pf-c-button--FontSize: var(--pf-c-chip__c-button--FontSize);
-
-    margin-top: var(--pf-c-chip__c-button--MarginTop);
-    margin-right: var(--pf-c-chip__c-button--MarginRight);
-    margin-bottom: var(--pf-c-chip__c-button--MarginBottom);
+  .pf-c-chip__content {
+    display: flex;
+    column-gap: var(--pf-c-chip__content--ColumnGap);
+    font-size: var(--pf-c-chip__content--FontSize);
+  }
+
+  .pf-c-chip__actions {
```

Comment:
can this be un-nested from the `.pf-c-chip` parent container? It's creating extra specificity.